client/src/components/DataFiles/DataFilesTable/DataFilesTable.js

```diff
@@ -23,13 +23,29 @@ const DataFilesTablePlaceholder = ({ section, data }) => {
   );
   const scheme = useSelector(state => state.files.params.FilesListing.scheme);
   const system = useSelector(state => state.pushKeys.target);
+  const currSystemHost = useSelector(
+    state =>
+      state.systems.definitions.list
+        .filter(sysDef => sysDef.id === state.files.params.FilesListing.system)
```

Comment:
It may be more straightforward to use the `.find` array method, as then we can determine whether or not there is a matching system: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find